xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -318,4 +292,47 @@ public void run() {
       }
     }
   }
+
+  /**
+   * Represents a successfully parsed and validated LoadBalancingConfig for XDS.
+   */
+  static final class XdsConfig {
+    private final String newBalancerName;
+    @Nullable
+    private final LbConfig childPolicy;
```

Comment:
`LbConfig` contains unparsed raw value. You should probably use a different type that is a `(policy, parsedConfig)` tuple. If you plan to do it later, please leave a TODO.